solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -229,6 +229,7 @@ export default {
                         );
                 } else {
                     await this.get_location_string()
+                    console.log("this.joined_locations", this.joined_locations);
```

Comment:
Do we want this console.log in here?  Doesnt matter either way to me, just wondering if you forgot it or something.

Reply:
whoops, lemme get rid of this.